statusmonitor/uves/control_test.go

```diff
@@ -0,0 +1,102 @@
+package uves
+
+import "testing"
+
+func Test_statusData_Status(t *testing.T) {
+	tests := []struct {
+		name    string
+		s       statusData
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "Single filled status",
+			s: [][]interface{}{
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "5",
+					},
+				},
+			},
+			want:    "5",
+			wantErr: false,
+		},
+		{
+			name: "Single empty status",
+			s: [][]interface{}{
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "",
+					},
+				},
+			},
+			want:    "0",
+			wantErr: false,
+		},
+		{
+			name: "Filled status with second string",
+			s: [][]interface{}{
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "5",
+					},
+					"ip-10.0.12.13-ec2-instance.internal",
+				},
+			},
+			want:    "5",
+			wantErr: false,
+		},
+		{
+			name:    "Empty status list",
+			s:       [][]interface{}{},
+			want:    "0",
+			wantErr: false,
+		},
+		{
+			name: "Too long status list",
+			s: [][]interface{}{
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "5",
+					},
+				},
+				{
+					map[string]string{
+						"@type": "u32",
+						"#text": "6",
+					},
+				},
+			},
+			want:    "",
```

Comment:
Make the zero value useful.
This line is redundant.

Reply:
Done, because I removed errors